script.js

```diff
@@ -1,5 +1,10 @@
 let deckElement = document.querySelector(".deck");
-const baseUrl = "https://api.scryfall.com/cards/search?q=name=";
+const baseUrl = "https://api.scryfall.com";
+const cardPath = "/cards/search?q=name=";
+const tokenPath = "/cards/named?exact=";
+
+function getCardUrl(cardName) { return baseUrl + cardPath + encodeURI(cardName); }
+function getTokenUrl(cardName) { return baseUrl + tokenPath + encodeURI(cardName); }
```

Comment:
Feel free to write:

    const getCardUrl = cardName => `${baseUrl}${cardPath}${encodeUri(cardName)}`;
    const getTokenUrl = cardName => `${baseUrl}${tokenPath}${encodeUri(cardName)}`;